From ca26b16732c2ad19b91aa59019d7092f64d111f3 Mon Sep 17 00:00:00 2001
From: Jerome Brunet <jbrunet@baylibre.com>
Date: Sun, 23 Jan 2022 19:37:26 +0100
Subject: [PATCH 2/2] meson: add initial libretech cc support

Add support for the libretech-cc based on the amlogic s905x

Signed-off-by: Jerome Brunet <jbrunet@baylibre.com>
---
 ucm2/conf.d/gx-sound-card/LIBRETECH-CC.conf | 1 +
 1 file changed, 1 insertion(+)
 create mode 120000 ucm2/conf.d/gx-sound-card/LIBRETECH-CC.conf

diff --git a/ucm2/conf.d/gx-sound-card/LIBRETECH-CC.conf b/ucm2/conf.d/gx-sound-card/LIBRETECH-CC.conf
new file mode 120000
index 000000000000..280ea38b04d7
--- /dev/null
+++ b/ucm2/conf.d/gx-sound-card/LIBRETECH-CC.conf
@@ -0,0 +1 @@
+../../Meson/p241/p241.conf
\ No newline at end of file
-- 
2.35.1

